test/Catch.hs

```diff
@@ -9,6 +10,11 @@ import Gen
 import Test.Tasty
 import Test.Tasty.Hedgehog
 
+tests :: TestTree
+tests = testGroup "Catch"
+  []
```

Comment:
I stubbed this in so that it’s easier to see that all of the effect test modules are being run in `Test.hs`.